src/components/layout/header.jsx

```diff
@@ -67,6 +69,23 @@ const Header = ({ className }) => {
                 : "opacity-0 pointer-events-none xl:opacity-100 xl:pointer-events-auto"
             )}
           >
+            {location.pathname !== "/" && (
+              <div className="group flex gap-1.5 items-center">
+                <a href="/" target="_blank" className="link ml-8 xl:ml-0" rel="noreferrer">
+                  The Leaderboard
+                </a>
+                <LinkIcon className="transform -rotate-45 before:w-[1.2rem] xl:before:w-[0.6rem] before:group-hover:w-[1.45rem] xl:before:group-hover:w-[0.75rem] translate-y-[-0.1rem]" />
+              </div>
+            )}
+
+            {location.pathname !== "/halloffame" && (
+              <div className="group flex gap-1.5 items-center">
+                <a href="/halloffame" target="_blank" className="link ml-8 xl:ml-0" rel="noreferrer">
+                  The Hall of Fame
+                </a>
+                <LinkIcon className="transform -rotate-45 before:w-[1.2rem] xl:before:w/[0.6rem] before:group-hover:w/[1.45rem] xl:before:group-hover:w/[0.75rem] translate-y-[-0.1rem]" />
+              </div>
+            )}
```

Comment:
Hall of fame isn't an external link therefore we won't need a redirection arrow and we could use the react router `Link` here to keep the navigation within the router. Also i feel that just `Hall of Fame` will be enough as the name